File: DevTools_Actuator_Lombok_yml/lombok_devtools/src/main/java/com/hexaware/springtools/entity/Student.java
package com.hexaware.springtools.entity;

import java.time.LocalDate;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
import lombok.extern.slf4j.Slf4j;



@AllArgsConstructor
@Setter
@Getter
@ToString
@Slf4j

@Table(name="Student_Details")
@Entity
public class Student {
	
		@Id
		private  int sid;
		private  String sname;
		private   LocalDate dob;
	
		public Student() {
			
			log.info("this is student default constructor");
		}
	
	
	

}

File: EcmaScript_TypeScript/TypeScript_demos/Student.ts


 export class  Student{

     private   sid:number;
      private  sname:string;

 
      public  constructor(sid , sname){

                this.sid = sid;
                this.sname = sname;

        }

      public  showStudentDetails(){


                console.log(this.sid +"  "+this.sname)


        }

        setSid(sid){


            this.sid = sid;
        }

        getSid(){

            return this.sid;
        }


}

let s1 = new Student(111,"king");


s1.showStudentDetails();